functional_tests/testdata/dotnet/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/dotnet/aspnet:7.0 AS base
```

Comment:
Not mandatory but I would recommend you go straight to 8 - 7 reaches EOL in May, the odd numbers are not LTS. Similar at line 5.

Reply:
Done